.expeditor/adhoc-canary.omnibus.yml

```diff
@@ -80,8 +80,8 @@ builder-to-testers-map:
     - ubuntu-18.04-x86_64
     - ubuntu-20.04-x86_64
     - ubuntu-22.04-x86_64
-  windows-2012r2-i386:
-    - windows-2012r2-i386
+  # windows-2012r2-i386:
+  #   - windows-2012r2-i386
```

Comment:
Small thing, but should we just delete the lines instead of commenting? We'll always have commit history.

Reply:
done.